transport_nantes/photo/static/photo/js/vote_system.js

```diff
@@ -0,0 +1,120 @@
+function getCookie(name) {
+    let cookieValue = null;
+    if (document.cookie && document.cookie !== '') {
+        const cookies = document.cookie.split(';');
+        for (let i = 0; i < cookies.length; i++) {
+            const cookie = cookies[i].trim();
+            // Does this cookie string begin with the name we want?
+            if (cookie.substring(0, name.length + 1) === (name + '=')) {
+                cookieValue = decodeURIComponent(cookie.substring(name.length + 1));
+                break;
+            }
+        }
+    }
+    return cookieValue;
+}
+
+
+/**
+ * Cast vote when the user has already voted on any photo
+ * @param {boolean} vote_nature "upvote" or "downvote"
+ * @param {HTMLElement} element "the button that has been pressed to cast vote"
+ */
+function castVote(vote_nature, element) {
+    $.ajax({
+        url: window.location.href,
+        headers: {"X-CSRFToken": getCookie("csrftoken")},
+        type: 'post',
+        data: {
+            vote_nature: vote_nature,
+        },
+        mode: 'same-origin',
+        success: () => {
+            console.log("Success");
```

Comment:
How will we make use of this?  How does this suffice to debug anything?

Reply:
It was dev sanity check haha, gotta remove it (as well as the "error" one)